src/Kanbas/Courses/Quizzes/Preview/Question/Common/ShortAnswers/index.tsx

```diff
@@ -0,0 +1,26 @@
+function ShortAnswers({ question }: { question: any }) {
+    return (
+        <div className="text">
+            <div className="question-text">
+                <p>{question.questionText}</p>
+            </div>
+            <div className="answers">
+                {question.correctAnswers.map((choice: any, index: number) => (
+                    <div className="answer" key={index}>
+                        <label className="d-flex justify-content-start">
+                            <span className="answer-input">
+                                <input
```

Comment:
Use React-Bootstrap components. The one to use here is `Form.Control` and `Form.Label` along with `Form.Group`